app/locale/vi.coffee

```diff
@@ -291,69 +291,69 @@ module.exports = nativeDescription: "Tiếng Việt", englishDescription: "Vietn
-#    anon_signup_ask_teacher: "Don't have one? Ask your teacher!"
-#    anon_signup_create_class: "Want to create a class?"
-#    anon_signup_setup_class: "Set up a class, add your students, and monitor progress!"
-#    anon_signup_create_teacher: "Create free teacher account"
     play_as: "Chơi là" # Ladder page
+    title: 'Chơi game cùng CodeCombat - Học Python, JavaScript, và HTML'
+    meta_description: 'Học lập trình thông qua chơi game dành cho người mới học. Em sẽ học Python và JavaScript thông qua việc giải mã mê cung, thiết kế game, và nâng cấp cho nhân vật. Em có thể chơi cùng bạn bè trên các đấu trường!'
+    level_title: '__level__ - Học lập trình Python, JavaScript, HTML'
+    video_title: '__video__ | Video'
+    game_development_title: '__level__ | Lập Trình Game'
+    web_development_title: '__level__ | Lập Trình Web'
+    anon_signup_title_1: "CodeCombat có"
+    anon_signup_title_2: "Phiên bản dành cho lớp học!"
+    anon_signup_enter_code: "Nhập Mật Khẩu lớp:"
+    anon_signup_ask_teacher: "Không có Mật Khẩu thì hỏi Giáo viên nhé!"
+    anon_signup_create_class: "Bạn cần mở lớp mới?"
+    anon_signup_setup_class: "Tạo bị phòng học, quản lý học sinh và theo dõi tiến độ học tập!"
+    anon_signup_create_teacher: "Tạo tài khoản Giáo viên miễn phí"
 #    get_course_for_class: "Assign Game Development and more to your classes!"
 #    request_licenses: "Contact our school specialists for details."
     compete: "Hoàn thành!" # Course details page
     spectate: "Quan sát" # Ladder page
 #    simulate_all: "Simulate All"
     players: "người chơi" # Hover over a level on /play
     hours_played: "Thời gian chơi" # Hover over a level on /play
-    items: "Trang Bị" # Tooltip on item shop button from /play
-    unlock: "Mua" # For purchasing items and heroes
+    items: "Đồ Nghề"  # Tooltip on item shop button from /play
+    unlock: "Mua"  # For purchasing items and heroes
     confirm: "Xác nhận"
-    owned: "Đã có" # For items you own
-    locked: "Bị khóa"
     available: "Khả dụng"
-    skills_granted: "Kỹ năng nhận được" # Property documentation details
-    heroes: "Tướng" # Tooltip on hero shop button from /play
-    achievements: "Thành Tích" # Tooltip on achievement list button from /play
-    settings: "Tùy Chỉnh" # Tooltip on settings button from /play
     poll: "Bỏ phiếu" # Tooltip on poll button from /play
-    next: "Tiếp" # Go from choose hero to choose inventory before playing a level
-    change_hero: "Đổi Tướng" # Go back from choose inventory to choose hero
-#    change_hero_or_language: "Change Hero or Language"
-    buy_gems: "Mua Ngọc"
-#    subscribers_only: "Subscribers Only!"
-#    subscribe_unlock: "Subscribe to Unlock!"
-#    subscriber_heroes: "Subscribe today to immediately unlock Amara, Hushbaum, and Hattori!"
-#    subscriber_gems: "Subscribe today to purchase this hero with gems!"
-    anonymous: "Người chơi ẩn danh"
+    owned: "Đã có"  # For items you own
```

Comment:
dont change order. `next:` should be on same line as before, just translate string instead.

Same for all below strings too

Reply:
This is the exact copy from [en.coffee](https://github.com/codecombat/codecombat/blob/master/app/locale/en.coffee#L331) I realized vi.coffee is a bit out of date compared with the English version, hence the difference and some changes. Do you want me to update it anyway?